health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/PopInbox.js

```
@@ -677,10 +703,25 @@ const isStatusConditionMet = (statusCount) => {
           style={{}}
         />}
 
+      {((!isRootApprover && isStatusConditionMet(totalStatusCount)) || disabledAction) &&
+        <ActionBar
+          actionFields={[
+            <Button label={t(`HCM_MICROPLAN_POP_INBOX_BACK_BUTTON`)} onClick={()=> {
+              history.push(`/${window.contextPath}/employee/microplan/select-activity?microplanId=${url?.microplanId}&campaignId=${url?.campaignId}`);
+            }} type="button" variation="primary" />,
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)